site/docs/build-javascript.md

```diff
@@ -36,7 +36,7 @@ following:
 
 You can either [Install Bazel](install.html) following the same steps that you
 would for backend development, or you can install NodeJS with npm and run
-`npm install -g @bazel/bazel`.
+`npm install -g @bazel/bazelisk` (and then invoke bazel via the `bazelisk` command).
```

Comment:
according to the `"bin"` entry in https://unpkg.com/@bazel/bazelisk@1.7.3/package.json you get a `bazel` binary from installing `@bazel/bazelisk`

so you don't need the parenthetical here. Just change @bazel/bazel to @bazel/bazelisk

Reply:
Done.